sdk/src/main/java/com/horizen/cryptolibprovider/CommonCircuit.java

```diff
@@ -0,0 +1,36 @@
+package com.horizen.cryptolibprovider;
+
+import com.horizen.provingsystemnative.ProvingSystem;
+import com.horizen.provingsystemnative.ProvingSystemType;
+import com.horizen.utils.BytesUtils;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class CommonCircuit {
+    private static final int maxSegmentSize = (1 << 18);
+
+    // Keys total max size values are the same as in MC
+    public static final int maxProofPlusVkSize = 9 * 1024;
```

Comment:
If we are following Java naming conventions for Java code, shouldn't public static final int fields be Upper Case with _ as separator (e.g. MAX_PROOF_PLUS_VK_SIZE)?